cali_mtx:
- 0.7496297221257042
- 0.6618564401958423
- 0.0011542430536478807
- 0.0
- -0.6618416917774864
- 0.7495987050785236
- 0.008207092646938913
- 0.0
- 0.00456669802530013
- -0.006916206755733453
- 0.9999656550868423
- 0.0
- 0.04139639376494078
- -0.09072127195790565
- 0.04342845033940606
- 1.0
child_frame_id: camera_link
parent_frame_id: panda_link8
